pkg/benchmark/README.md

```diff
@@ -0,0 +1,202 @@
+# pod and container lifecycle benchmark for kata
```

Comment:
could we move this into kata repo instead?

Reply:
Just as @gnawux said, it is not suitable to place a kata specific README here, thus I remove it.

Reply:
Thanks, other parts lgtm